src/brouwers/shop/models/utils.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+
+def get_max_order() -> int:
+    from .payments import PaymentMethod
```

Comment:
Why this import is not top level?

Reply:
the `get_max_order` name is imported in the `payments` module, so you'd get a circular import otherwise